thirdparty/fetch-geoip-db.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Die on any error for Travis CI to automatically retry:
+set -e
+
+download_dir="${0%/*}/download"
+mkdir -p "${download_dir}"
+cd "${download_dir}"
+
+filename="GeoLite2-Country.mmdb"
+
+if [ -e $filename ]; then
+	if [ -n $(find . -name $filename -mtime +30 -print) ]; then
```

Comment:
It would be clearer if you inverted this test, and then removed and downloaded inside this block instead of exiting.